### ISSUE TITLE
Set the output power

### ISSUE THREAD

**Issue by wshe1** [bastibl/gr-ieee802-11#131]
Hello Bastian:
       I am using USRP B210 and want to set a fixed output power. In your paper,the SDR we adjust the gain and the signal amplitude,assuming a maximum output power of 20dBm, which is listed as a typical value in the datasheet.My question is whether the input power of the USRP B210 is also 10dBm in your program's default configuration.

**Comment by bastibl** [comment]
No, the config in the flow graph is supposed to use the dynamic range as efficient as possible. I manually adjusted the amplitude so that the signal uses the interval [-1;1] without too much clipping.

However, the output of the PHY is normalized to an average power of 1, so you have to adjust the multiplier accordingly. (Note that even then, that's only a rough estimate of the power. The USRP is not calibrated and the actual output power might vary with frequency and sample rate.)
